WISE-Community/WISE
# AT "Go Home" should properly remove from current authors list

This is for #1298

How to test:
1. Open a project in AT.
2. In another window, log in as admin user, and verify that the project is being edited in the "View Projects Currently Being Authored" page.
3. Click on "Go Home" in account menu in the AT.
4. The user/project should no longer be listed in the admin's "View Projects Currently Being Authored" page.

Also verify that:
1. the user is taken to the teacher homepage instead of the main homepage.
2. this works regardless of how you open the project: via "edit" link in the project library, or via choosing project from project listing in the AT.
